Description: make chromedriver snap-aware, so that it looks for the right chromium binary
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/chrome/test/chromedriver/chrome/chrome_finder.cc
+++ b/chrome/test/chromedriver/chrome/chrome_finder.cc
@@ -52,6 +52,7 @@ void GetApplicationDirs(std::vector<base
 void GetApplicationDirs(std::vector<base::FilePath>* locations) {
   // TODO: Respect users' PATH variables.
   // Until then, we use an approximation of the most common defaults.
+  locations->push_back(base::FilePath("/snap/bin"));
   locations->push_back(base::FilePath("/usr/local/sbin"));
   locations->push_back(base::FilePath("/usr/local/bin"));
   locations->push_back(base::FilePath("/usr/sbin"));
